### DISCUSSION TITLE
v3.0-beta3

### DISCUSSION THREAD

**Discussion by tomakino** [tomakino/CSLyric#47] [Announcements]
- 修复`Apple music`和 `酷狗音乐`歌词错误
- 修复一些系统歌词颜色反色错误

<hr /><em>This discussion was created from the release <a href='https://github.com/tomakino/CSLyric/releases/tag/v3.0-beta3'>v3.0-beta3</a>.</em>

**Comment by eeidjbs** [comment]
网易云没有魅族状态栏歌词开关，使用不了![Screenshot_2025-05-01-12-34-04-372_com.netease.cloudmusic.jpg](https://github.com/user-attachments/assets/0b9c9a36-48b5-46c2-a35e-9064ccba2b33)

😭😭😭
